src/style.css

```
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir-Heavy;
```

```suggestion
  font-family: Avenir;
```
it should be the same font-family, but different  font-weight